lang/extract_json_strings.py

```diff
@@ -1208,121 +1208,126 @@ def extract(item, infilename):
         else:
             writestr(outfile, name, **kwargs)
         wrote = True
-    if "name_suffix" in item:
-        writestr(outfile, item["name_suffix"], **kwargs)
-        wrote = True
-    if "name_unique" in item:
-        writestr(outfile, item["name_unique"], **kwargs)
-        wrote = True
-    if "job_description" in item:
-        writestr(outfile, item["job_description"], **kwargs)
-        wrote = True
-    if "use_action" in item:
-        extract_use_action_msgs(outfile, item["use_action"], item.get("name"),
-                                kwargs)
-        wrote = True
-    if "conditional_names" in item:
-        for cname in item["conditional_names"]:
-            c = "Conditional name for {} when {} matches {}".format(
-                name, cname["type"], cname["condition"])
-            writestr(outfile, cname["name"], comment=c, format_strings=True,
-                     pl_fmt=True, **kwargs)
+    def do_extract(item):
+        nonlocal wrote
```

Comment:
You can return `wrote` from `do_extract` instead of using a nonlocal variable.

Reply:
done